tests/tests.js

```diff
@@ -1,33 +1,157 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const {
+  afterEach,
+  beforeEach,
+  describe,
+  it
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
+chai.use(sinonChai)
+const {
+  expect
+} = chai
+
+describe('formatPrice', () => {
+  let spyOnPrice
+  let spyOnformatPrice
+  let sandbox
+
+  foreEach(() => {
+    sandbox = sinon.createSandbox()
+
+    priceSpy = sandbox.spy(pricing, 'price')
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+  })
+  afterEach(() => {
+    sandbox.restore()
+    it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {
+      const x = 15.1566663
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
+  })
+
+  describe('calculatesLTDPrice', () => {
+    it('returns the price for a disability product for an employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee']
+      }
+      const price = pricing.calculateLTDPrice(product.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+  })
+
+  it('returns the LTD price minus the employer contribution', () => {
     const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      familyMembersToCover: [ee]
     }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+    const price = pricing.calculateLTDPrice(products.voluntaryLife, employee, selectedOptions)
+  })
```

Comment:
This test is now floating on its own instead of being inside a describe as it should be.